api/src/test/java/org/cardanofoundation/rosetta/api/block/mapper/BlockTxToRosettaTransactionTest.java

```diff
@@ -37,7 +37,8 @@
 import static org.cardanofoundation.rosetta.common.util.Constants.ADA;
 import static org.cardanofoundation.rosetta.common.util.Constants.ADA_DECIMALS;
 
-class BlockTxToRosettaTransactionTest extends BaseMapperTest {
+class
+BlockTxToRosettaTransactionTest extends BaseMapperTest {
```

Comment:
I bet it's not intentional. As well as 3 blank lines after assertions :)

Reply:
Done